src/lib/components/MainPanel.js

```diff
@@ -11,13 +11,14 @@ export default class MainPanel extends React.Component {
     super(props);
     this.state = {
       datepickerActive: false,
-      dateChoice: null
+      dateChoice: null,
+      defaultDateChoice: moment()
```

Comment:
We'll need to change this to `props.moment()`, as I suspect you know.
(And there are a couple of other `moment`s down below that will need changing, too.)